### PR TITLE
Proxy config

### PR DESCRIPTION
**Pull request by jonathanlbt1**
Adding configuration to run openlineage-proxy directly from within its folder with docker-compose.yml file. 

I also edited the README file to add a description of how to use it. It will make it a lot quicker for anybody to have a go at it and see the application running.

### CODE REVIEW

File: proxy/backend/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ RUN ./gradlew --no-daemon shadowJar
 FROM eclipse-temurin:11
 WORKDIR /usr/src/app
 COPY --from=build /usr/src/app/build/libs/openlineage-proxy-*.jar /usr/src/app
-COPY proxy.dev.yml proxy.dev.yml
```
Comment (pawel-big-lebowski) [inline proxy/backend/Dockerfile:18]: Can we keep it `proxy.dev.yml` to indicate it is an example setup to be run locally?

File: proxy/backend/Dockerfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ RUN ./gradlew --no-daemon shadowJar
...
+COPY proxy.yml proxy.yml
 COPY docker/entrypoint.sh entrypoint.sh
-EXPOSE 5000 5001
+EXPOSE 6000 6001
```
Comment (pawel-big-lebowski) [inline proxy/backend/Dockerfile:20]: any particular reason for this change?

**Review by pawel-big-lebowski — COMMENTED** [review]
Thank you for refreshing proxy setup. 
It's been a while since modifying this part. 
I left few minor comments. Let me know what do you think.

File: proxy/backend/Dockerfile (hunk 1 above)
Comment (jonathanlbt1) [reply proxy/backend/Dockerfile:18]: Sure thing! I'll include the changes in this file but I'll leave them commented with some instructions so people know how to proceed.

**Review by pawel-big-lebowski — APPROVED** [review]

File: proxy/backend/Dockerfile (hunk 2 above)
Comment (jonathanlbt1) [reply proxy/backend/Dockerfile:20]: Yes, Marquez is configured to work on port 5000, so I changed this port so one can use Marquez to give it a try.

**Pull request merged**